src/components/address-finder/index.njk

```diff
@@ -0,0 +1,86 @@
+---
+title: Address finder
+---
+{% from "views/partials/example/_macro.njk" import patternlibExample %}
+{% from "components/external-link/_macro.njk" import onsExternalLink %}
+{% from "components/panel/_macro.njk" import onsPanel %}
+
+
+The address finder component uses the ONS Address Index API to search for and return addresses. 
+
+## When to use this component
+
+Use the address finder when you need users to find and confirm a UK address. This could be their own or any other address that is required for the question.
+
```

Comment:
Similar to the recent changes we made to the Input component, I think we should reference the pattern here. So add a line line "There is specific guidance on how to ask for [addresses](url)"